scripts/fast-tune.sh

```
@@ -68,6 +68,54 @@ case "$APP" in
     # listing order.
     PROFILE_MATCH=replicaset-mariadb-client
     ;;
+  flux-source-controller)
+    NS=flux-system
+    SUITE=example/flux-source-controller-attacks.yaml
+    FUNCTESTS=example/flux-source-controller-functional-tests.yaml
+    PROFILE_MATCH=source-controller-.*-manager
```

Shellcheck (0.11.0)

[warning] Brace expansions and globs are literal in assignments. Quote it or use an array.

(SC2125)